.github/workflows/build.yaml

```diff
@@ -27,9 +27,20 @@ jobs:
       - name: Update pip
         run: pip3 install pip --upgrade
```

Comment:
So this is relevant ony for CS9, right?

Reply:
I still think so, I know there were some pip installation problems without it.
I'll create one testing commit without CS9

Reply:
Looks like it was related only to python36, both passed the installation even without the upgrade.
Removing